src/OpenTok/OpenTok.php

```diff
@@ -478,6 +478,62 @@ public function forceDisconnect($sessionId, $connectionId)
         return $this->client->forceDisconnect($sessionId, $connectionId);
     }
 
+    /**
+     * force a publisher of a specific stream in an OpenTok session to mute its audio.
+     *
+     * @param string $sessionId The OpenTok session ID containing the stream.
+     *
+     * @param string $streamId The stream ID.
+     *
+     * @return bool Whether the call succeeded or failed.
+     */
+    public function forceMuteStream(string $sessionId, string $streamId): bool
+    {
+        Validators::validateSessionId($sessionId);
+        Validators::validateStreamId($streamId);
+
+        try {
+            $this->client->forceMuteStream($sessionId, $streamId);
+        } catch (\Exception $e) {
+            return false;
+        }
+
+        return true;
+    }
+
+    /**
+     * Force all streams (except for an optional list of streams) in an OpenTok session
+     * to mute published audio.
+     *
+     * @param string $sessionId The OpenTok session ID.
+     *
+     * @param array<string> $options This array defines options and includes the following keys:
+     *
+     * <ul>
+     *    <li><code>'excludedStreamIds'</code> (array, optional) &mdash; An array of stram IDs
+     *    corresponding to streams that should not be muted. This is an optional property.
+     *    If you omit this property, all streams in the session will be muted.</li>
+     *
```

Comment:
The property name for the `POST` data expected by the [API endpoint](https://tokbox.com/developer/rest/#force_mute_session) is `excludedStreams`, not `excludedStreamIds`.